public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -251,17 +251,17 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
         </Button>
       ) : null}
 
-      {existing ? (
+      {existing && isCortexLokiOrRecordingRule(watch) && (
         <Button
           variant="secondary"
           type="button"
           onClick={() => setShowEditYaml(true)}
           disabled={submitState.loading}
           size="sm"
         >
-          {isCortexLokiOrRecordingRule(watch) ? 'Edit YAML' : 'Export'}
+          {'Edit YAML'}
```

Comment:
nit: Don't think we need this wrapped with `{' '}` anymore :)

Reply:
ops! you're right!